static/js/hotkey.js

```diff
@@ -426,6 +426,10 @@ export function process_enter_key(e) {
             return true;
         }
 
+        if (overlays.invite_open() && $(e.target).attr("id") === "custom-expiration-time-input") {
+            return true;
+        }
+
```

Comment:
I would expect it to be better to do a more fine-grained key handler for this particular element rather than adding code here; we do this for `enter` in a lot of other components.